.github/workflows/release.yml

```diff
@@ -4,9 +4,8 @@ on:
   push:
     branches:
       - master
-  release:
-    types:
-      - published
+    tags:
+      - '*'
```

Comment:
Should we make this `v*` in a half-hearted attempt to match semver tags? I don't know of another way to do it right now

Reply:
I think it'd be best to keep this as is for now. I don't imagine we'd be publishing any tags other than of the form `v1.2.3`, and this would keep our options open should we decide to change our version scheme at some point.

Reply:
meh, went ahead with the change